src/containers/Landing.tsx

```diff
@@ -9,100 +9,102 @@ import { RouterHistory } from "@sentry/react/types/reactrouter";
-	const theme = useTheme();
+export default function Landing({ navigate }: { navigate: RouterHistory }) {
+  const theme = useTheme();
 
-	const config = {
-		showCube: false,
-		dimension: '3D',
-		velocity: 1,
-		boundaryType: 'bounce',
-		antialias: false,
-		direction: {
-			xMin: 0.2,
-			xMax: 1,
-			yMin: 0,
-			yMax: 1,
-			zMin: -1,
-			zMax: 1
-		},
-		lines: {
-			colorMode: 'solid',
-			color: '#d7ddc8',
-			transparency: 0.5,
-			limitConnections: true,
-			maxConnections: 20,
-			minDistance: 150,
-			visible: true
-		},
-		particles: {
-			colorMode: 'solid',
-			color: '#7b1fce',
-			transparency: 0.9,
-			shape: 'circle',
-			boundingBox: 'canvas',
-			count: 500,
-			minSize: 10,
-			maxSize: 75,
-			visible: false
-		},
-		cameraControls: {
-			enabled: true,
-			enableDamping: true,
-			dampingFactor: 0.2,
-			enableZoom: true,
-			autoRotate: true,
-			autoRotateSpeed: 0,
-			resetCameraFlag: false
-		}
-	}
+  const config = {
+    showCube: false,
+    dimension: "3D",
+    velocity: 1,
+    boundaryType: "bounce",
+    antialias: false,
+    direction: {
+      xMin: 0.2,
+      xMax: 1,
+      yMin: 0,
+      yMax: 1,
+      zMin: -1,
+      zMax: 1,
+    },
+    lines: {
+      colorMode: "solid",
+      color: "#d7ddc8",
+      transparency: 0.5,
+      limitConnections: true,
+      maxConnections: 20,
+      minDistance: 150,
+      visible: true,
+    },
+    particles: {
+      colorMode: "solid",
+      color: "#7b1fce",
+      transparency: 0.9,
+      shape: "circle",
+      boundingBox: "canvas",
+      count: 500,
+      minSize: 10,
+      maxSize: 75,
+      visible: false,
+    },
+    cameraControls: {
+      enabled: true,
+      enableDamping: true,
+      dampingFactor: 0.2,
+      enableZoom: true,
+      autoRotate: true,
+      autoRotateSpeed: 0,
+      resetCameraFlag: false,
+    },
+  };
 
-	return (
-		<div>
-			<div className="landing-page-nav" style={{ alignItems: "center", height: '10vh' }}>
-				<img
-					src={logo}
-					alt="Pareto"
-					height="40"
-					width="178"
-					style={{ marginTop: 10, marginLeft: 12 }}
-				/>
-				<div>
-					<Link
-						to="/signup"
-						style={{ color: "white", marginTop: 18, marginRight: 18 }}
-					>
-						{I18n.get("signup")}
-					</Link>
-					<Link
-						to="/login"
-						style={{ color: "white", marginTop: 18, marginRight: 18 }}
-					>
-						{I18n.get("login")}
-					</Link>
-				</div>
-			</div>
-			<div style={{ height: '90vh' }} id="animation-container" >
-
-				<ParticleField config={config} />
-				<div id="overlay">
-					<h2>Build the future</h2>
-					<Link
-						className="btn"
-						to="/signup"
-						style={{ color: "white", marginTop: 18, marginRight: 18 }}
-					>
-						{I18n.get("signup")}
-					</Link>
-					<Link
-						className="btn"
-						to="/login"
-						style={{ color: "white", marginTop: 18, marginRight: 18 }}
-					>
-						{I18n.get("login")}
-					</Link>
-				</div>
```

Comment:
lol I didn't know the linting was so messed up, thanks for fixing it up.

Reply:
you are welcome